src/site/content/en/fast/codelab-cloudinary-url-api/index.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: codelab
+title: Optimize images and serve next-gen formats using Cloudinary
+authors:
+  - ahmadawais
+date: 2019-06-24
+description: |
+  In this codelab, learn how to use Cloudinary with its URL API to optimize JPEG and
+  PNG images for faster download and serve next-gen formats.
+glitch: codelab-cloudinary?path=index.html:25:0
```

Comment:
I got the codelab reviewed at a local meetup. 8 out of 11 attendees had problems figuring out how to open `index.html` since it was there first time with Glitch and small screens glitch hides the left Files Explorer. 

Then I worked around a way for the codelab to open up with the exact file instead `index.html`. This helped them get started quickly.

So, I'd like to request to keep `?path=index.html:25:0` part there for better Developer Experience.

Thoughts?

Reply:
That sounds reasonable to me.

@robdodson - Can you confirm that adding URL params to the glitch name won't break anything?

Reply:
I think you can use the path param in YAML to do this:

```
glitch: codelab-cloudinary
path: index.html:25:0
```

Reply:
@robdodson thanks for sharing that. Fixed in db09fa3